ts/Core/Pointer.ts

```diff
@@ -1272,7 +1272,7 @@ class Pointer {
             pEvt.returnValue = false;
         }
 
-        if (chart.mouseIsDown === 'mousedown') {
+        if (chart.mouseIsDown === 'mousedown' || this.touchSelect(pEvt)) {
             this.drag(pEvt);
         }
```

Comment:
Can we use this as an opportunity to do `e = this.normalize(e)` instead of creating pEvt?

Reply:
Okay, I see now that it is in type sense not working.